examples/pointer.rs

```diff
@@ -173,11 +180,11 @@ impl InputManagerHandler for InputManager {
 fn main() {
     init_logging(L_DEBUG, None);
     let cursor = Cursor::create(Box::new(ExCursor));
-    let xcursor_theme = XCursorTheme::load_theme(None, 16).expect("Could not load theme");
+    let xcursor_manager = XCursorManager::create(Some("default".to_string()), 24).expect("Could not create xcursor manager");
```

Comment:
Thanks to the `Into<Option<String>>`, you shouldn't have to write `Some("default".to_string())`, only `"default".to_string()`